### DISCUSSION TITLE
Which colors to use for a QInput in a QToolbar?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#9584] [General - Components / Directives / etc]
I have a QToolbar  with some QButtons ("SORT BY MODIFIED" below) and I wanted to add a QInput (Search" below) to the bar:

![image](https://user-images.githubusercontent.com/1345886/120934593-33072b00-c6ff-11eb-9c2f-f1d8908fe4db.png)

While the QButton colors are fine (without declaring any), the ones of the QInput are blended into the QToolbar. I would like them to be white, how can I enforce this?

**Comment by ontwikkelfabriek** [comment]
@wsw70 

`class="text-white"`

See the docs `Using as CSS Classes`:
https://quasar.dev/style/color-palette

**Comment by rstoenescu** [comment]
You're using it on a dark background, so first of all add the 'dark' prop.

**Reply by wsw70** [reply]
Thanks, adding `dark` helped a bit:

![image](https://user-images.githubusercontent.com/1345886/120995605-d7cc4b80-c785-11eb-81ba-c0cd694910b9.png)

This corresponds to the following code:

```
  <q-header class="bg-primary text-white">
      <q-toolbar>
        <q-input model-value="undefined" dense v-model="search" placeholder="Search" class="text-white" dark>
          <template v-slot:append>
            <q-btn flat icon="cancel" @click="search=''" class="text-white"/>
          </template>
          <template v-slot:before>
            <q-icon name="search" class="text-white"/>
          </template>
        </q-input>
(...)
```

So `dark` lightened up the text, but not at the level of `text-white` that worked on the `before` and `after`  icons 

(I also added a `<q-btn>`  in the screenshot to show that its text is white by default without any styling - which is understandable as it has a `primary` background and the text color is handled appropriately (by setting it to white by default)) .
